release_collection.py

```diff
@@ -280,8 +285,10 @@ def update_galaxy_version(args, galaxy, versions_updated):
         and any(versions_updated[0:3])
         and not args.no_auto_version
     ):
-        galaxy_ver = StrictVersion(galaxy["version"])
-        major, minor, micro = galaxy_ver.version
+        galaxy_ver = lsr_parse_version(galaxy["version"])
+        major = galaxy_ver.major
```

Comment:
```python
         major, minor, micro = galaxy_ver.release
```

Reply:
Sorry, I missed it... (I tested with python3.6, but there was no problem reported... (;_;) I wish it failed there...):

Reply:
> Sorry, I missed it... (I tested with python3.6, but there was no problem reported... (;_;) I wish it failed there...):

How did you test with python 3.6?  When I test, I'm doing something like this:
```
podman run --rm -it centos:7 bash
yum -y install python3-setuptools
python3
from pkg_resources import parse_version
....
```

Reply:
> > Sorry, I missed it... (I tested with python3.6, but there was no problem reported... (;_;) I wish it failed there...):
> 
> How did you test with python 3.6? When I test, I'm doing something like this:

I see... I just ran release_collection.py with python3.6 on my laptop.
```
$ python3.6
Python 3.6.15 (default, Jun 10 2022, 00:00:00) 
```
Maybe, that was not good enough... I'm going to run the tests on podman from now on... Thanks!